src/components/common/ConfirmModal.tsx

```
@@ -37,7 +37,7 @@ export default function ConfirmModal({ isOpen, message, onConfirm, onClose }: Co
                     <div className="flex flex-col gap-4 justify-center items-center">
                         <p className="text-xl font-medium">{message}</p>
                         <div className="flex gap-4">
-                            <button className="px-4 py-2 rounded-md bg-main-color text-secondary-color-dark" onClick={onConfirm}>확인</button>
+                            <button className="px-4 py-2 rounded-md bg-main-color text-white" onClick={onConfirm}>확인</button>
                             <button className="px-4 py-2 rounded-md bg-cancel-background text-white" onClick={onClose}>취소</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)